android_10/src/org/ros/android/MasterChooser.java

```diff
@@ -348,6 +372,50 @@ protected boolean isQRCodeReaderInstalled(Intent intent) {
   }
 
   /**
+   * Stores the recent Master URI to preferences. This implementation does not use
```

Comment:
nit: rewording. Perhaps something like: "adds the given URI to the list of recent Master URIs shared preferences". And perhaps the method name should be `addRecentMasterURI` instead.